c2corg_ui/static/img/fonts/howTo.md

```diff
@@ -0,0 +1,32 @@
+# Creating font from SVG files
```

Comment:
It would make more sense to rename this file to README.md => it would then be automatically displayed in this directory in github.